flowauth/backend/tests/test_access_reflects_server_limits.py

```diff
@@ -2,17 +2,17 @@
 
 import datetime
 
+import pytest
+
 from flowauth.models import (
-    Group,
-    GroupServerPermission,
-    GroupServerTokenLimits,
     Server,
-    ServerCapability,
     User,
     db,
+    Role,
 )
 
 
+@pytest.mark.skip(reason="No groups anymore; not sure if this is relevant")
```

Comment:
I think if it's no longer relevant we should remove it instead of skipping.

Reply:
presumably still relevant but for user's instead of groups?

Reply:
Still relevant for roles, I think.